qa/automation/CareerPortal.Tests.Scenarios/Send your CV/Send_your_CV.feature

```diff
@@ -0,0 +1,146 @@
+Feature: 1.1.2 Send_CV.feature
+
+Background: 
+	Given user accessed Quantori Career Portal from his browser
+	And user scrolled down to CV sending form
+
+Scenario Outline: 1.1.2_01 User can successfully send CV
+	Given user inputted "<Name>", "<Email>", "<Checkbox>", "<Message>", attached <CV> in "Attach CV"
```

Comment:
It should be divided in several lines and you should also provide names of the fields where you put info: 
Given user entered <name> into "Name" field
And user entered <email> into "Email" field and so on